gno.land/pkg/gnoclient/integration_test.go

```diff
@@ -21,8 +23,14 @@ import (
 )
 
 func TestCallSingle_Integration(t *testing.T) {
-	// Set up in-memory node
-	config, _ := integration.TestingNodeConfig(t, gnoenv.RootDir())
+	// Setup packages
+	rootdir := gnoenv.RootDir()
+	config := integration.TestingMinimalNodeConfig(gnoenv.RootDir())
+	meta := loadpkgs(t, rootdir, "gno.land/r/demo/deep/very/deep")
```

Comment:
Why do we load this?

Reply:
We need this package below to make some test calls.